src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorLanguageFeatureBase.ts

```diff
@@ -31,14 +36,24 @@ export class RazorLanguageFeatureBase {
                     ? razorDocument.csharpDocument
                     : razorDocument.htmlDocument;
 
-                const synchronized = await this.documentSynchronizer.trySynchronizeProjectedDocument(
-                    document,
-                    projectedDocument,
-                    languageResponse.hostDocumentVersion,
-                    token);
-                if (!synchronized) {
-                    // Could not synchronize
-                    return null;
+                if (languageResponse.hostDocumentVersion == this.undefinedDocumentVersion) {
+                    // There should always be a document version attached to an open document.
+                    // Log it and move on as if it was synchronized.
+                    if (this.logger.verboseEnabled) {
+                        this.logger.logVerbose(
+                            `Could not find a document version associated with the document '${getUriPath(document.uri)}'.`);
+                    }
+                }
```

Comment:
nit: else should be on same line as the `}`. We should update the tslint.json to capture this.